app/templates/secrets.hbs

```diff
@@ -9,7 +9,7 @@
         <span class="icon icon-menu">menu</span>
       {{/link-to}}
       <form action="#">
-        {{input placeholder="Search..." value=query}}
+        {{input id="search" placeholder="Search..." value=query}}
```

Comment:
According to [this Ember documentation](http://guides.emberjs.com/v1.10.0/views/customizing-a-views-element/) it seems to be OK to override the `id` parameter of elements of the templates, it looks like Ember will use it.